src/app/shared/components/consultation-card/consultation-card.component.html

```diff
@@ -10,7 +10,7 @@
           <p class="responses-text">
             {{consultation?.consultationResponsesCount ? consultation?.consultationResponsesCount : ''}}
             <span  *ngIf="consultation?.consultationResponsesCount == 1">
-              {{'Response so far' | translate}}
+              {{'Resp so faronse' | translate}}
```

Comment:
The text doesn't  look right, is this intentional?

Reply:
No bro, by mistake